airgun/entities/architecture.py

```diff
@@ -28,10 +28,8 @@ class ShowAllArchitectures(BaseNavigator):
     VIEW = ArchitectureView
 
     def step(self, *args, **kwargs):
-        menu_click(
-            ["//a[@id='hosts_menu']", self.view.page_navigate_locator],
-            self.view.browser
-        )
+        # TODO: No prereq yet
+        self.view.navigation.select_item('hosts_menu', 'menu_item_architectures')
```

Comment:
How about defining some dictionary of correspondence with UI inside the widget? So we could use it in pretty clear for any user way like `self.view.navigation.go_to('Hosts', 'Architectures')`?

Reply:
I only used IDs because you used them in the original code. I prefer using text matching. As we do in CF automation :)

Reply:
Original code wasn't even a widget, just a temporary stub. Yes, text match would probably work as well.

Reply:
Okay, so I will replace it with text and make sure it has exactly the same interface as the verstical navigation in patternfly